server/directory.go

```diff
@@ -0,0 +1,20 @@
+package server
+
+import "github.com/coniks-sys/coniks-go/merkletree"
+
```

Comment:
I might add a typedef ``type directory merkletree.PAD`` here. Then we can make ``Update()`` a function of ``directory``. The server policies and signing key could be passed as parameters?